src/Cryptor.ts

```diff
@@ -1,131 +1,160 @@
-import * as fs from 'fs';
-import * as crypto from 'crypto';
-import * as process from 'process';
+import * as fs from 'fs'
+import * as crypto from 'crypto'
+import * as process from 'process'
+import { spawn, Worker, ModuleThread } from 'threads'
+import { CryptorWorker } from './CryptorWorker'
 
 // TODO: function docs
 
+// interface CryptorParameters {
```

Comment:
These sorts of comments should be left out. If commented code is left in our master branch, there must be a good a reason for doing so. Consider this:

1. If they are not used anymore - delete them
2. If they are to be implemented in the future - turn it into an issue and delete them
3. If they are not understood - create an issue and delete them
4. If they are documentation - then it should be written in a doc format